src/database/Persona/Persona.db.ts

```diff
@@ -347,9 +347,20 @@ export async function queryProfilesDB(
     return result
 }
 
+const fuse = new Fuse([] as ProfileRecord[], {
+    shouldSort: true,
+    threshold: 0.45,
+    maxPatternLength: 32,
+    minMatchCharLength: 1,
+    keys: [
+        { name: 'nickname', weight: 0.8 },
+        { name: 'identifier.network', weight: 0.2 },
+    ],
+})
+
 export async function queryProfilesPagedDB(
     options: {
-        after: ProfileIdentifier | undefined
+        after?: ProfileIdentifier
```

Comment:
There is no difference between calling it with `undefined` or just omitted totally. The only thing the user should remember is when the `after` field absents they will get profiles from the top. Otherwise, they will get the rest profiles after the given one.